doc/wave_calib.rst

```diff
@@ -181,20 +181,26 @@ quit the GUI to see if you want to save the solution. Note,
 you can increase this tolerance using the command line option
 `pixtol`, or by setting the `force_save` command line option.
 
-To use this wavelength solution in your reduction, you will
-need to add your solution to the PypeIt database. To do this,
-you will need to move the output file into the master directory,
-which will be similar to the following directory:
-
-``/directory/to/PypeIt/pypeit/data/arc_lines/reid_arxiv/name_of_your_solution.fits``
-
-Once your solution is in the database, you will be able to
-run PypeIt in the standard :ref:`wvcalib-fulltemplate` mode.
-Make sure you add the following line to your pypeit file::
-
-  [calibrations]
-     [[wavelengths]]
-        reid_arxiv = name_of_your_solution.fits
+In addition to writing the wavelength solution to the file
+``wvarxiv.fits`` in the current working directory, ``PypeIt``
+now also saves the solution in the PypeIt cache and prints
+a message indicating how to use it, such as:
+
+   .. code-block:: console
+
+      [INFO]    :: Your arxiv solution has been written to ./wvarxiv.fits
+      [INFO]    :: Your arxiv solution has also been cached.
+                  To utilize this wavelength solution, insert the
+                  following block in your PypeIt Reduction File:
+                  [calibrations]
+                     [[wavelengths]]
+                        reid_arxiv = manual_ldt_deveny_20220422T1006.fits
```

Comment:
I've addressed this in 2 ways.
  1. No longer writing to generic `wvarxiv.fits`, but instead writing to `wvarxiv_{spec}_{datetime}.fits` in the local directory.
  2. I've corrected the text above this code block to indicate what is saved in the cache is labeled with the spectrograph and timestamp of creation for uniqueness.  This is in case a user is doing manual IDs for multiple setups and wants to be able to use all of them.